### ISSUE TITLE
[error] [client] Caught an unhandled exception: failed to open file 'C:/Users/xxx/AppData/Local/multipass-client-certificate/multipass_cert_key.pem': Permission denied(13)

### ISSUE THREAD

**Issue by mariojoseee** [canonical/multipass#3081]
**Describe the bug**
multipass won't launch. conditions using windows 11 home and have installed virtualbox

**To Reproduce**
How, and what happened?
1. multipass set local.driver=virtualbox
[error] [client] Caught an unhandled exception: failed to open file 'C:/Users/xxx/AppData/Local/multipass-client-certificate/multipass_cert_key.pem': Permission denied(13)

2. multipass launch
[error] [client] Caught an unhandled exception: failed to open file 'C:/Users/xxx/AppData/Local/multipass-client-certificate/multipass_cert_key.pem': Permission denied(13)

**Expected behavior**
the expected result is that multipass launch can run

**Logs**
Please provide logs from the daemon, see [accessing logs](https://multipass.run/docs/accessing-logs) on where to find them on your platform.

**Additional info**
 - OS: [Windows 11 Home Single Language]
- `multipass version 1.11.1+`
- `multipass info --all`
[error] [client] Caught an unhandled exception: failed to open file 'C:/Users/xxx/AppData/Local/multipass-client-certificate/multipass_cert_key.pem': Permission denied(13)
- `multipass get local.driver`
[error] [client] Caught an unhandled exception: failed to open file 'C:/Users/xxx/AppData/Local/multipass-client-certificate/multipass_cert_key.pem': Permission denied(13)

**Additional context**
After I finished installing multipass 1.11.1+ there is the message below :
Attempting to set driver to 'virtualbox'
[error] [client] Caught an unhandled exception: failed to open file 'C:/Users/xxx/......
Failed to set driver. Will try again in 1 second

**Comment by sharder996** [comment]
Hey @mariojoseee, the client certificate is located within your user's home directory and should by default have the same permissions as you. However, as a workaround are you able to manually navigate to the location of the specified file and give yourself full permissions?

**Comment by mariojoseee** [comment]
Sorry, I don't know. How to manually navigate the location of the specified file and give full permissions?

**Comment by sharder996** [comment]
Take the address `C:/Users/xxx/AppData/Local/multipass-client-certificate` replacing "xxx" with your username and paste it in the address bar of the file explorer. If you right-click on the file `multipass_cert_key.pem` you can verify the permissions by selecting the security tab from the properties window and clicking on your user under "Group or user names". If your user does not appear or is denied any permissions, you can click "Edit" and then modify permissions or add permissions by clicking "Add" > entering your username in the textbox > "OK" > then modify permissions by checking the Allow checkboxes.

Alternatively, open an administrator privileged powershell instance and execute the following commands to give yourself access to the file
```
$filePath = "C:/Users/xxx/AppData/Local/multipass-client-certificate"
$identity = [System.Security.Principal.WindowsIdentity]::GetCurrent()
$acl = Get-Acl -Path $filePath
$acl.SetOwner($identity.User)
$acl.AddAccessRule((New-Object System.Security.AccessControl.FileSystemAccessRule($identity.User, "FullControl", "Allow")))
Set-Acl -Path $filePath -AclObject $acl -Recurse
```
Don't forget to replace "xxx" with your username.

**Comment by mariojoseee** [comment]
Before I did allow to modify permission, I saw that the initial settings were already checked in the checkbox. The settings for these three users and groups are the same. Do I need to check the special permissions?
![image](https://github.com/canonical/multipass/assets/94614259/adb7f391-c6e6-416d-b431-84df55cfbb1c)


For an alternative way that uses powershell, is it really like this? I get a red message like this
![image](https://github.com/canonical/multipass/assets/94614259/de406e22-b37a-4ecc-8a0f-87f0a47ef092)

@sharder996
